### ISSUE TITLE
Icon spacing

### ISSUE THREAD

**Issue by KevinBatdorf** [ovk/brackets-simple-todo#2]
No clue why (conflict with another plugin?), but here is what I see:
![screen shot 2015-01-29 at 2 07 45 am](https://cloud.githubusercontent.com/assets/1478421/5944867/d4b80448-a75c-11e4-9148-637c1cadd434.png)

**Comment by ovk** [comment]
@KevinBatdorf, could you please tell what other plugins do you have installed?

**Comment by KevinBatdorf** [comment]
Sure, here's that I have:
```
adobe.brackets.extract
bigeyex.brackets-sftp-upload
brackets-beautify
brackets-bower
brackets-code-folding
brackets-emmet
brackets-quick-markup
brackets-scss-lint
brackets-terminal
brackets-wakatime
brackets-wp-functions-hint
camden.caniuse
camden.csslint
camden.htmlhint
camden.jshint
custom-work-for-brackets
dalcib.brackets-gulp
dannymoerkerke.brackets-codeintel
dnbard.extensions-rating
globexdesigns.brackets-jscs
globexdesigns.brackets-jsxhint
hirse.outline-list
insya.tools
ivogabe.icons
open-in-browser
os.xcode.theme
ovk.brackets-simple-todo
quickdocsphp
talmand.select-lines
zaggino.brackets-git

**Comment by ovk** [comment]
The issue is fixed in v0.1.2

**Comment by KevinBatdorf** [comment]
Perfect, thanks!
